src/styles/variables.json

```diff
@@ -1,110 +1,124 @@
 {
-    "white": "#fff",
```

Comment:
Did anything in this file change? Can we avoid this large diff?

Reply:
Yes, I've added 2 variables in [src/styles/constants/_layout.scss](https://github.com/box/box-ui-elements/pull/2904/files#diff-a53963adad1686c603addefb455563c7244331db4bef3dd85399c29401105cea) and I believe that "variables" files are auto generated. I haven't made any changes to them manually.